### PR TITLE
Fix Hudi docs layout

### PR DESCRIPTION
**Pull request by devozerov**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hudi.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ To use Hudi, we need:
 * Network access from the Presto coordinator and workers to the distributed object storage.
 * Access to a Hive metastore service (HMS).
 * Network access from the Presto coordinator to the HMS. Hive metastore access with the Thrift
-protocol defaults to using port 9083.
+  protocol defaults to using port 9083.
```
Comment (ajaygeorge) [inline presto-docs/src/main/sphinx/connector/hudi.rst:22]: does this fix the docs?
Can you please add a before and after screenshot for the docs in the test plan

**Review by ajaygeorge — COMMENTED** [review]
Added request for Test plan

**Comment by ajaygeorge** [comment]
@devozerov Can you please update the test plan when you get a chance.

**Review by steveburnett — COMMENTED** [review]
This is already fixed in master, see https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/hudi.rst . 

Screenshot of local build of master: 
Screenshot 2024-07-08 at 1 35 38 PM

Suggest closing this.

**Comment by steveburnett** [comment]
Closing due to lack of activity and age. If this is still relevant, please re-open this PR, rebase, and resolve any file conflicts.

**Pull request closed without merging**